pkgs/os-specific/linux/kernel/common-config.nix

```diff
@@ -562,6 +562,11 @@ let
       KEYS_REQUEST_CACHE               = whenAtLeast "5.3" yes;
       # randomized slab caches
       RANDOM_KMALLOC_CACHES            = whenAtLeast "6.6" yes;
+
+      # NIST SP800-90A DRBG
+      CRYPTO_DRBG_HASH                 = whenAtLeast "3.10" yes;
+      CRYPTO_DRBG_CTR                  = whenAtLeast "3.10" yes;
```

Comment:
Drop the `whenAtLeast`s — we don't support kernels that old.

Reply:
I'd probably add a comment saying why it's enabled, too, given the NIST's questionable history.

Reply:
Done for both